app/objects/c_ability.py

```diff
@@ -64,7 +65,7 @@ def unique(self):
     def __init__(self, ability_id, tactic=None, technique_id=None, technique=None, name=None, test=None,
                  description=None, cleanup=None, executor=None, platform=None, payloads=None, parsers=None,
                  requirements=None, privilege=None, timeout=60, repeatable=False, buckets=None, access=None,
-                 variations=None, language=None, code=None, build_target=None, **kwargs):
+                 variations=None, language=None, code=None, build_target=None, hidden=False, **kwargs):
```

Comment:
Check out the get_variations function in the master branch to fix merge conflicts